Documentation/asciidoc/feat/build_aosp.adoc

```diff
@@ -0,0 +1,277 @@
+## Build x86 AOSP emulator with custom kernel config
+
+### Prerequisites
+
+* Linux x86_64 host
+* 256GB of free SSD space
+* 64GB of RAM
+* https://github.com/BurntSushi/ripgrep[`ripgrep`] | `grep`
+* https://developer.android.com/tools/adb?hl=de[`adb`]
+* repo (`sudo apt-get install repo`)
+
+NOTE: export `adb` from installed Android Tools (e.g. `<path>/Android/Sdk/platform-tools/adb`) instead of installing via `apt` (Ubuntu).
+
+### Commands
+
+.check whether tracing syscalls is enabled on an Android device
+[source,bash]
+----
+# adb -e shell -- zcat /proc/config.gz| grep CONFIG_BPF_SYSCALL
```

Comment:
Das muss noch geändert werden auf `adb -e shell -- zcat /proc/config.gz| grep CONFIG_FTRACE_SYSCALLS`

Reply:
done in #68